lib/ssl/src/ssl_cipher.erl

```diff
@@ -1097,24 +1084,20 @@ rsa_signed(_) ->
        (_) -> false
     end.
 %% Cert should be signed by RSA
-rsa_signed_suites(Ciphers, Version) ->
-    filter_suites(Ciphers, #{key_exchange_filters => [rsa_signed(Version)],
+rsa_signed_suites(Ciphers) ->
+    filter_suites(Ciphers, #{key_exchange_filters => [rsa_signed()],
                              cipher_filters => [],
                              mac_filters => [],
                              prf_filters => []}).
-ecdsa_signed({3,N}) when N >= 3 ->
```

Comment:
@IngelaAndin the code here is not semantically the same, I guess this is my error and not correct?